### DISCUSSION TITLE
ALL my POC 'Public' Flowise Chatbots via links generated in Flowise have started duplicating their responses.

### DISCUSSION THREAD

**Discussion by BarOfSoapOnWednesday** [FlowiseAI/Flowise#2854] [Q&A]
Looked into the configuration and other options and triled changing things. Can't seem to find the reason. Help? Not sure why this is so, it is occuring for ALL my chatbots.

**Comment by BarOfSoapOnWednesday** [comment]
have also updated fork several times. no difference.

**Reply by SHornoiu** [reply]
Hi, I have the same issue after the last update!
Help?

**Comment by BarOfSoapOnWednesday** [comment]
This appears to be fixed. Thanks!
